CryptoTechReminderSystem.Test/GetLateDevelopersTests.cs

```diff
@@ -0,0 +1,249 @@
+            private SlackGatewayStub _slackGatewayStub;
+            
+            [SetUp]
+            public void SetUp()
+            {
+                _harvestGatewayStub = new HarvestGatewayStub()
+                {
+                    Developers = new[]
+                    {
+                        new HarvestDeveloper()
+                        {   
+                            Id = 1337,
+                            FirstName = "Fred",
+                            LastName = "Flintstone",
+                            Email = "fred@fred.com",
+                        },
+                        new HarvestDeveloper()
+                        {
+                            Id = 123,
+                            FirstName = "Joe",
+                            LastName = "Bloggs",
+                            Email = "Joe@Bloggs.com"
+                        },
+                    }
+                };
+                
+                _slackGatewayStub = new SlackGatewayStub()
+                {
+                    Developers = new[]
+                    {
+                        new SlackDeveloper
+                        {
+                            Email = "fred@fred.com",
+                            Id = "U8723",
+                        }, 
+                        new SlackDeveloper
+                        {
+                            Email = "Joe@Bloggs.com",
+                            Id = "U9999",
+                        }
+                    }
+                };
+            }
+            
+            [Test]
+            public void CanGetLateDevelopers()
+            {
+                _harvestGatewayStub.TimeSheets = new[]
+                {
+                    new TimeSheet()
+                    {
+                        Hours = 7,
+                        UserId = 1337
+                    }, 
+                    new TimeSheet()
+                    {
+                        Hours = 7,
+                        UserId = 1337
+                    }, 
+                    new TimeSheet()
+                    {
+                        Hours = 7,
+                        UserId = 1337
+                    }, 
+                    new TimeSheet()
+                    {
+                        Hours = 7,
+                        UserId = 1337
+                    }, 
+                    new TimeSheet()
+                    {
+                        Hours = 7,
+                        UserId = 1337
+                    }, 
+                };
+                
+                var getDevelopers = new GetLateDevelopers(_slackGatewayStub, _harvestGatewayStub);
+                
+                var response = getDevelopers.Execute();
+         
+                response.Developers.First().Should().Be("U9999");
+            }
+            
+            [Test]
+            public void CanGetAnotherLateDeveloper()
+            {                
+                _harvestGatewayStub.TimeSheets = new TimeSheet[]
```

Comment:
Similarly, we can do:

```csharp
_harvestGatewayStub.TimeSheets = Enumerable.Repeat(
    new TimeSheet { Hours = 7, UserId = 1337 }, 5
).ToArray();
```

Reply:
Good change, implemented in [f883a50](https://github.com/chubberlisk/crypto_tech/pull/9/commits/f883a5093730c1459bcf0da990d9387f5080e3f1)